src/helm/proxy/clients/huggingface_model_registry.py

```diff
@@ -28,6 +30,9 @@ class HuggingFaceModelConfig:
 
     If None, use the default revision."""
 
+    path: Optional[str] = None
+    """Local path to the Hugging Face model weights"""
```

Comment:
Maybe add the default path here: `./huggingface_models`

Reply:
Thanks! I added a [comment](https://github.com/stanford-crfm/helm/blob/c1a088a812e4a9ab4164d4432878d1d44090d926/src/helm/proxy/clients/huggingface_model_registry.py#L35-L40) explaining how this parameter is set and what it means:)

Reply:
I don't set it directly to the local path by default (like `path: str = ./huggingface_models`) because `None` means to load it from the hub instead.